### DISCUSSION TITLE
How to supress warning about offline mode?

### DISCUSSION THREAD

**Discussion by sealab2066** [ansible/ansible-lint#3153] [General]
When i run ansible-lint i see this warning
```
WARNING  Skipped installing collection dependencies due to running in offline mode.
```

Here is my ansible-lint config
```
---
profile: production
skip_list:
  - template-instead-of-copy
  - fqcn-builtins
  - command-shell
  - yaml
  - no-jinja-nesting
  - name[template]
  - no-tabs
  - fqcn
  - role-name[path]
  - var-naming
warn_list: []
parseable: true
offline: true
```

How to avoid this warning, without using "-q" key?
I need standart output like as 
```
ansible-lint
WARNING  Skipped installing collection dependencies due to running in offline mode.

Passed with production profile: 0 failure(s), 0 warning(s) on 2118 files.
```
With "Passed with prod...", but without "WARNING bla bla" and i can't delete `offline: true` from config of course

**Comment by MarkusTeufelberger** [comment]
The code that emits this warning is in a different repository: https://github.com/ansible/ansible-compat/blob/f51159bb7954e7353c62264aa8cdc4e44e2096e7/src/ansible_compat/runtime.py#L347-L350 - maybe ask there to reduce the logging/warning level to info?
